### DISCUSSION TITLE
Question related challenges

### DISCUSSION THREAD

**Discussion by shrivrat0809** [Cyfrin/foundry-full-course-cu#225] [Q&A] [answered]
Where can I find the qr code to solve challenges after each lesson ? I am unable to find it. Please help me anyone !!!

**Comment by Shiva-Sai-ssb** [comment]
Hello @shrivrat0809 , you will find two QR codes on the video after completing each lesson. One is for Sepolia, and the other is for Arbitrum. These QR codes will be displayed directly on the YouTube video. Here is a sample one after completing lesson one.

![Screenshot (28)](https://github.com/Cyfrin/foundry-full-course-f23/assets/112751524/04550030-2937-4abc-812f-34dcf76ca768)

**Comment by asobiesk** [comment, marked as answer]
See #135 for the full list of challenges.
